fud/icarus/icarus.py

```diff
@@ -167,11 +173,17 @@ def cleanup(tmpdir: SourceType.Directory):
 
         # Schedule
         tmpdir = mktmp()
+        data_path = get_verilog_data()
+        data_path_exists: bool = (
+            config.get(["stages", "verilog", "data"]) or
+            config.get(["stages", "mrxl", "data"])
```

Comment:
Again, it's odd that this stage has to understand what `mrxl.data` is in order to work correctly. Is there a way to hide `mrxl.data`'s existence from this stage?